.github/workflows/enos-test-ui.yml

```diff
@@ -0,0 +1,107 @@
+---
+name: Vault UI Tests
+
+on:
+  workflow_call:
```

Comment:
Shouldn't this be `workflow_dispatch` since we want to run it from the Actions tab?

Reply:
I guess maybe we want both, since we'll call this from the nightly but probably want to make executing it super easy for a branch.

Reply:
Yeah I like that idea... done.